wallet/how-to/use-non-evm-networks/starknet-snap/index.md

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_position: 1
+sidebar_label: Starknet
+description: Learn about how Starknet Snap connects to MetaMask using `get-starknet`.
+tags:
+  - Snaps
+  - non-EVM
+---
+
+# Build with Starknet
+
+Starknet is a Layer 2 network built on Ethereum. It processes transactions off the main Ethereum chain to increase speed and reduce costs, while still benefiting from Ethereum's security.
+
+One major aspect of Starknet development is integrating with user wallets. There are two primary approaches to achieve this integration:
+
+- [Using direct Snap invocation with `wallet_invokeSnap](#use-wallet_invokesnap-directly)
+- [Using the `get-starknet` integration](#use-the-get-starknet-integration)
+
+Both methods enable developers to connect Starknet dapps with wallet solutions. 
+The choice between them depends on the specific needs of the project, the desired level of control, and familiarity with Starknet and Snap technologies.
+
+## About `get-starknet`
+
+`get-starknet` simplifies Starknet network interactions by extending MetaMask functionality. It eliminates the need for custom wallet logic and allows you to connect to Starknet and perform transactions through the MetaMask interface.
+The `get-starknet` toolkit works with the Starknet Snap to enable interaction between dapps and the Starknet network through MetaMask. 
+Starknet Snap is a MetaMask extension designed specifically to allow users to manage Starknet accounts and perform transactions on the Starknet network. 
+When you integrate `get-starknet` into your dapp, it creates a `WalletAccount` object. `WalletAccount` acts as a connection between dapps and MetaMask and provides a structured way to manage Starknet-specific interactions. 
+This allows users to send transactions, sign messages, and manage accounts within the MetaMask interface and it can be extended to multiple wallets, with Starknet capabilities added on.
```

Comment:
```suggestion
`get-starknet` is a library that simplifies Starknet network interactions.
`get-starknet` works with the Starknet Snap to extend the functionality of MetaMask and enable dapps to interact with users' Starknet accounts in MetaMask.

When you integrate `get-starknet` into your dapp, it creates a `WalletAccount` object. `WalletAccount` acts as a connection between dapps and MetaMask and provides a structured way to manage Starknet interactions.
This allows users to send Starknet transactions, sign Starknet messages, and manage Starknet accounts within MetaMask, and this functionality can be extended to multiple wallets.
```

Suggest re-wording – this page needs to be shortened in general. Also let's use one term consistently to refer to the `get-starknet`: tool, toolkit, library, integration, etc?